packages/api/src/controllers/insightController.js

```diff
@@ -141,20 +145,24 @@ const insightController = {
       try {
         const farmID = req.params.farm_id;
         const dataPoints = await knex.raw(
-          `SELECT f.grid_points, SUM(CASE WHEN fc.deleted = false and fc.end_date >= NOW() THEN 1 ELSE 0 END) as count
+          `SELECT area.grid_points, SUM(CASE WHEN fc.deleted = false and fc.end_date >= NOW() THEN 1 ELSE 0 END) as count
           FROM "field" f
+          JOIN "location" on location.location_id = f.location_id
+          JOIN "figure" on figure.location_id = location.location_id
+          JOIN "area" on figure.figure_id = area.figure_id
           LEFT JOIN "fieldCrop" fc
-          ON fc.field_id = f.field_id
-          WHERE f.farm_id = ?
-          AND f.deleted = false
-          GROUP BY f.grid_points`, [farmID]);
+          ON fc.location_id = f.location_id
+          WHERE location.farm_id = ?
+          AND location.deleted = false
+          GROUP BY area.grid_points`, [farmID]);
```

Comment:
Sorry Jimmy, could we make this query only query farm_site_boundaries for now 
I think that would require a JOIN before the location join.